wave_containers/include/wave/containers/measurement_container.hpp

```diff
@@ -61,8 +61,8 @@ struct measurement_container {
     struct sensor_key : member<T, decltype(T::sensor_id), &T::sensor_id> {};
 
     // Define a composite key which combines the above two keys
-    // This lets us search elements sorted by sensor id first, then time
-    struct sensor_and_time_key : composite_key<T, sensor_key, time_key> {};
+    // This lets us search elements sorted by time first, then  sensor id
+    struct sensor_and_time_key : composite_key<T, time_key, sensor_key> {};
```

Comment:
Probably. But see the example above. It is very plausible that multiple measurements are inserted with the same time; at least, the library should expect it.

Note that is irrelevant to the choice of ordering the index by time first, or by sensor first.